frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/capacity-and-nodes.tsx

```diff
@@ -89,10 +91,23 @@ export const EnableTaintNodes: React.FC<EnableTaintNodesProps> = ({ state, dispa
 
   return (
     <Checkbox
-      label={t('ceph-storage-plugin~Enable taint nodes')}
-      description={t(
-        "ceph-storage-plugin~Selected nodes will be dedicated to OpenShift Container Storage's use only",
-      )}
+      label={
+        <>
+          {t('ceph-storage-plugin~Mark nodes as dedicated')}{' '}
+          <Tooltip
+            content={
+              <Trans t={t} ns="ceph-storage-plugin">
+                This will taint the nodes with the
+                <code>key: node.ocs.openshift.io/storage</code>, <code>value: true</code>, and&nbsp;
+                <code>effect: NoSchedule</code>
```

Comment:
I doubt if this is accented by translator. Can you check once if this giving missing i18n key error in browser console? If not, then lets try this:
```suggestion
                <code>key: node.ocs.openshift.io/storage</code>, <code>value: true</code>, and{' '}
                <code>effect: NoSchedule</code>
```